test/e2e/e2e_test.go

```diff
@@ -38,7 +38,7 @@ var (
 
 	communityOperators = flag.String(
 		"communityOperators",
-		"quay.io/operator-framework/upstream-community-operators@sha256:098457dc5e0b6ca9599bd0e7a67809f8eca397907ca4d93597380511db478fec",
+		"quay.io/operatorhubio/catalog:latest",
```

Comment:
@joelanford unfortunately the specific SHAs for this image are not kept around for very long, as there are no long-lived tags. We will have to suffice with `:latest`, which might break tests if they are asserting on specifics.

Reply:
Seems fine for now. It'd be nice if we did the thing we are now doing in [catalogd](https://github.com/operator-framework/catalogd/blob/a1663ec2b64f560330ce57f78a8018357d400656/Makefile#L150-L152) and [operator-controller](https://github.com/operator-framework/operator-controller/blob/5cbc9a0d5e3bbeab3ce200a9fd1da66453f0247b/Makefile#L154-L165) where we have all of our test artifacts actually come out of the repo.

Definitely not a blocker for this though, IMO.